packages/editor-ui/src/api/templates.ts

```diff
@@ -42,10 +43,15 @@ export async function getWorkflows(
 	apiEndpoint: string,
 	query: { skip: number; limit: number; categories: number[]; search: string },
 	headers?: IDataObject,
-): Promise<{ totalWorkflows: number; workflows: ITemplatesWorkflow[] }> {
+): Promise<{
+	totalWorkflows: number;
+	workflows: ITemplatesWorkflow[];
+	filters: TemplateSearchFacet[];
+	out_of: number;
+}> {
 	return get(
 		apiEndpoint,
-		'/templates/workflows',
+		'/templates/search',
 		{
 			skip: query.skip,
 			rows: query.limit,
```

Comment:
how does this map to the page parameter typesense expects? https://typesense.org/docs/0.24.0/api/search.html#pagination-parameters

Reply:
This is mapped on the backend:
`per_page` --> `rows`
`page` --> `skip`